Holovibes/sources/analysis/tools_analysis_debug.cc

```diff
@@ -0,0 +1,101 @@
+#include "tools_analysis_debug.hh"
+#include "cuda_memory.cuh"
+
+float* load_CSV_to_float_array(const std::filesystem::path& path)
+{
+    std::string filename = path.string();
+    std::ifstream file(filename);
+
+    if (!file.is_open())
+    {
+        std::cerr << "Erreur : impossible d'ouvrir le fichier " << filename << std::endl;
+        return nullptr;
+    }
+
+    std::vector<float> values;
+    std::string line;
+
+    // Lire le fichier ligne par ligne
+    while (std::getline(file, line))
+    {
+        std::stringstream ss(line);
+        std::string value;
+        // Lire chaque valeur séparée par des virgules (ou espaces, selon le fichier)
+        while (std::getline(ss, value, ','))
+        {
+            try
+            {
+                values.push_back(std::stof(value)); // Convertir la valeur en float et l'ajouter au vecteur
+            }
+            catch (const std::invalid_argument&)
+            {
+                std::cerr << "Erreur de conversion de valeur : " << value << std::endl;
+            }
+        }
+    }
```

Comment:
too much comment in the language of Moliere
Maybe remove this function if useless